xLights/VideoReader.cpp

```diff
@@ -841,7 +841,7 @@ AVFrame* VideoReader::GetNextFrame(int timestampMS, int gracetime)
         //same frame, just return
         return _dstFrame;
     }
-    if (timestampMS >= timeOfPrevFrame && timestampMS < currenttime) {
+    if (timestampMS >= timeOfPrevFrame - 1 && timestampMS < currenttime) {
```

Comment:
Would be useful to have a comment here so we know why the "-1" is needed.

Reply:
So it's because of the pervasive, awful assumption that the frame rate can be captured as an integer milliseconds per frame.  With the exception of 20, 25, 40, and a few others, it can't, and those are hardly ever used in videos... lots of 24 and 30 and so on there.  So timeOfPrevFrame, in ms, is potentially off by 1... no harm in giving it a fudge factor of 1.

Reply:
Yeah I meant add a comment in the code.  A year from now a developer running into this won't know why its there.

Reply:
A comment in the xLights code?  AYS?
(I will happily put that comment in.)

Reply:
Yes a comment right above that line.  But it can be shorter...something like "subtracting -1 from previous frame due to integer roundoff issues for certain frame rates".  I'll merge it right after.